src/tslint-cli.ts

```diff
@@ -175,14 +175,14 @@ const processFile = (file: string) => {
     }
 };
 
-const fileOrFiles = argv.f;
+let files = [];
+if (argv.f instanceof Array) {
+    files = files.concat(argv.f);
+} else if (typeof argv.f === "string") {
+    files.push(argv.f);
+}
+files = files.concat(argv._);
 
-if (typeof fileOrFiles === "string") {
-    processFile(fileOrFiles);
-} else {
-    for (const ix in fileOrFiles) {
-        if (fileOrFiles.hasOwnProperty(ix)) {
-            processFile(fileOrFiles[ix]);
-        }
-    }
+for (const file of files) {
```

Comment:
use `let`

This feels like a bug in the TS compiler? `file` shouldn't be allowed to be a `const` since its value changes on every iteration of the loop...